### PR TITLE
add compile using docker doc

### PR DESCRIPTION
**Pull request by abby-cyber**
https://github.com/vesoft-inc/nebula-docs-cn/issues/2939

### CODE REVIEW

**Review by MegaByte875 — APPROVED** [review]
LGTM

**Review by cooper-lzy — APPROVED** [review]

File: docs-2.0/4.deployment-and-installation/2.compile-and-install-nebula-graph/7.compile-using-docker.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+
+  这将会将 NebulaGraph 的源代码克隆到当前目录的`nebula`子目录中。
+
+## 步骤
+
+
+1. 拉取 NebulaGraph 编译镜像。
+
+  ```bash
+  docker pull vesoft/nebula-dev:ubuntu2004
```
Comment (wey-gu) [inline docs-2.0/4.deployment-and-installation/2.compile-and-install-nebula-graph/7.compile-using-docker.md:26]: could consider 2204 if there is one :)

File: docs-2.0/4.deployment-and-installation/2.compile-and-install-nebula-graph/7.compile-using-docker.md [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+  docker network create nebula-net
+  ```
+
+3. 运行编译容器。
+
+  现在，我们可以在 Docker 容器中编译 NebulaGraph。执行以下命令：
+
+  ```bash
+  docker run -ti \
+    --network nebula-net \
```
Comment (wey-gu) [inline docs-2.0/4.deployment-and-installation/2.compile-and-install-nebula-graph/7.compile-using-docker.md:45]: 这里是假设了之后这个服务可以和已有的容器的 NebulaGraph cluster 一起跑的，可以说明一下，创建网络不是必须的。

**Review by wey-gu — APPROVED** [review]

File: docs-2.0/4.deployment-and-installation/2.compile-and-install-nebula-graph/7.compile-using-docker.md (hunk 1 above)
Comment (abby-cyber) [reply docs-2.0/4.deployment-and-installation/2.compile-and-install-nebula-graph/7.compile-using-docker.md:26]: 2004 is the latest.

File: docs-2.0/4.deployment-and-installation/2.compile-and-install-nebula-graph/7.compile-using-docker.md (hunk 2 above)
Comment (abby-cyber) [reply docs-2.0/4.deployment-and-installation/2.compile-and-install-nebula-graph/7.compile-using-docker.md]: I think the `--network` flag can be deleted in this case

Comment (wey-gu) [reply docs-2.0/4.deployment-and-installation/2.compile-and-install-nebula-graph/7.compile-using-docker.md]: Yes, that'll be better :)

**Pull request merged**